ui/src/app/utils/cdr-versions.tsx

```diff
@@ -39,9 +41,12 @@ function hasDefaultCdrVersion(
 // does not consider tier; IDs are globally unique, enforced by the API DB
 function findCdrVersion(
   cdrVersionId: string,
-  cdrTiers: CdrVersionTiersResponse
+  cdrTiers: CdrVersionTiersResponse | CdrVersionStore
```

Comment:
these types only differ in that `tiers` may be optional in one or the other.  Adding types clarified that we were calling this function with both.